docs/user_manual/components.md

```diff
@@ -657,9 +657,9 @@ $$
 ## Fault
 
 * type name: `fault`
-* * base: {hoverxreftooltip}`user_manual/components:base`
+* base: {hoverxreftooltip}`user_manual/components:base`
 
-`fault` defines a short circuit location in the grid. At this moment a fault can only happen at a `node`.
+`fault` defines a short circuit location in the grid. At the time of writing, a fault can only happen at a `node`.
```

Comment:
Not just at the time of writing; For now, a fault can only happen at a node. Maybe just change it in "A fault can only happen at a `node`."